pandas/core/generic.py

```diff
@@ -7687,17 +7689,35 @@ def resample(
             By default the input representation is retained.
         loffset : timedelta, default None
             Adjust the resampled time labels.
+
+            .. deprecated:: 1.1.0
+                You should add the loffset to the `df.index` after the resample.
+                like this:
+                ``df.index = df.index.to_timestamp() + to_offset(loffset)``
+                (a more complete example is present below)
+
         base : int, default 0
             For frequencies that evenly subdivide 1 day, the "origin" of the
             aggregated intervals. For example, for '5min' frequency, base could
             range from 0 through 4. Defaults to 0.
+
+            .. deprecated:: 1.1.0
+                The new arguments that you should use are 'offset' or 'origin'.
+                ``df.resample(freq="3s", base=2)``
+                becomes
+                ``df.resample(freq="3s", offset="2s")``
+
         on : str, optional
             For a DataFrame, column to use instead of index for resampling.
             Column must be datetime-like.
-
         level : str or int, optional
             For a MultiIndex, level (name or number) to use for
             resampling. `level` must be datetime-like.
+        origin : pd.Timestamp, default None
+            The timestamp on which to adjust the grouping. If None is passed,
```

Comment:
add versionadded 1.1

Reply:
done in the last commit on every added argument on the public API.